ioctl/cmd/ws/wsprojectregister.go

```diff
@@ -0,0 +1,59 @@
+package ws
+
+import (
+	"math/big"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/iotexproject/iotex-core/ioctl/cmd/ws/contracts"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+)
+
+var wsProjectRegisterCmd = &cobra.Command{
+	Use: "register",
+	Short: config.TranslateInLang(map[config.Language]string{
+		config.English: "register w3bstream project",
+		config.Chinese: "创建w3bstream项目",
+	}, config.UILanguage),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		out, err := registerProject(int64(projectID.Value().(uint64)))
+		if err != nil {
+			return output.PrintError(err)
+		}
+		output.PrintResult(output.JSONString(out))
+		return nil
+	},
+}
+
+func init() {
+	projectID.RegisterCommand(wsProjectRegisterCmd)
+	_ = wsProjectRegisterCmd.MarkFlagRequired(projectID.Label())
+
+	transferAmount.RegisterCommand(wsProjectRegisterCmd)
+
+	wsProject.AddCommand(wsProjectRegisterCmd)
+}
+
+func registerProject(projectID int64) (any, error) {
+	caller, err := NewContractCaller(projectRegistrarABI, projectRegistrarAddr)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create contract caller")
+	}
+	caller.SetAmount(big.NewInt(int64(transferAmount.Value().(uint64))))
```

Comment:
uint64 is not enough for token amount in rau unit, which can only represent around 18 IOTX at most

Reply:
has this been fixed?